### PR TITLE
done third task

### PR DESCRIPTION
**Pull request by BarsTK**

### CODE REVIEW

File: task3/task3.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<!-- perform binary search in sorted number array-->
+  <meta charset="utf-8">
+  <script type="text/javascript">
+    function binSearch(a, b) {
```
Comment (NikitaSedyx) [inline task3/task3.html]: just curious, why do you put your script in html?

File: task3/task3.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      var x = a.split(',');
```
Comment (NikitaSedyx) [inline task3/task3.html]: hm, a should be array

**Comment by BarsTK** [comment]
I took note of the comments and removed main function to separate file.
About html: to you could check function without console. Do I need another
way?

**Comment by NikitaSedyx** [comment]
@BarsTK 
your binSearch should take sorted array as first parameter
in execBin.. you can transform your string to array... it's simple

File: task3/task3.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  <meta charset="utf-8">
+  <script src="search.js"></script>
+  <script type="text/javascript">
+    function execBinSearch() {
+      var arrIn = document.getElementById("values").value;
+      var arr = arrIn.split(',');
+      var val = document.getElementById("number").value;
+      const result = binSearch(arr, +val);
+      
+      alert(result == -1 ? "Element is not found" :"Searched number is in position with index = " + result);
```
Comment (NikitaSedyx) [inline task3/task3.html]: nit: ` : ` spaces

File: task3/search.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+function binSearch(arr, searchNumb) {
+  var len = arr.length;
+  const ascFlag = (arr[0] < arr[len - 1]);
```
Comment (NikitaSedyx) [inline task3/search.js]: there is practise to name constants NAME_OF_CONSTANT upper snake case

File: task3/search.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  const diffIdxPos = 1;
+  var leftIdx = 0;
+  var rightIdx = len - 1;
+  var searchedIdx = -1;
```
Comment (NikitaSedyx) [inline task3/search.js]: if you use const you can use let

File: task3/search.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+function binSearch(arr, searchNumb) {
+  const DIFF_IDX_POS = 1;
+  var len = arr.length;
+  var leftIdx = 0;
+  var rightIdx = len - 1;
```
Comment (NikitaSedyx) [inline task3/search.js]: I mean if you use const you can use let instead of var. and not let instead of const :)

**Pull request merged**